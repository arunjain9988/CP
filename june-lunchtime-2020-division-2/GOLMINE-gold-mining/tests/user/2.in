1
2
20 14 4
10 2 5