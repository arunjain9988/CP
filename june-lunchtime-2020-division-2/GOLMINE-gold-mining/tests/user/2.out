12.85714 7.14285